### PR TITLE
Fetch contact list from Nomis

### PR DESCRIPTION
**Pull request by alan**

### CODE REVIEW

File: app/decorators/visit_decorator.rb [hunk 1]
```diff
@@ -7,6 +7,9 @@ class VisitDecorator < Draper::Decorator
     :prisoner_availability_unknown?,
     :slot_availability_unknown?,
     :slots_unavailable?,
+    :contact_list_unknown?,
```
Comment (alan) [inline app/decorators/visit_decorator.rb]: @chrishanes these are the methods you can use from the view with the `@visit` object

File: app/services/nomis/contact.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    attribute :restrictions, Array[ContactRestriction]
+
+    def approved?
+      approved_visitor
+    end
+
+    def active?
+      active
+    end
+
```
Comment (alan) [inline app/services/nomis/contact.rb]: @chrishanes methods on each contact that you can use to populate the banned form fields

File: config/application.rb [hunk 3]
```diff
@@ -108,5 +108,8 @@ class Application < Rails::Application
 
     config.public_prisons_with_slot_availability =
       ENV['PUBLIC_PRISONS_WITH_SLOT_AVAILABILITY']&.split(',')&.map(&:strip) || []
+
+    config.staff_prisons_with_nomis_contact_list =
+      ENV['STAFF_PRISONS_WITH_NOMIS_CONTACT_LIST']&.split(',')&.map(&:strip) || []
```
Comment (StupidCodeFactory) [inline config/application.rb]: There seem to be an emerging pattern here, a encapsulating this behaviour in a united tested class might not have been a bad idea.

File: app/services/prisoner_contact_list.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+class PrisonerContactList
+  delegate :approved, to: :contact_list
```
Comment (StupidCodeFactory) [inline app/services/prisoner_contact_list.rb]: that's a bit convoluted. I was looking for that method missed this lines a few times.

**Pull request merged**